src/glib-networking.mk

```diff
@@ -0,0 +1,25 @@
+PKG             := glib-networking
+$(PKG)_WEBSITE  := https://www.gnome.org
+$(PKG)_DESCR    := Network-related GIO modules for glib
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 2.54.1
+$(PKG)_CHECKSUM := eaa787b653015a0de31c928e9a17eb57b4ce23c8cf6f277afaec0d685335012f
+$(PKG)_SUBDIR   := glib-networking-$($(PKG)_VERSION)
+$(PKG)_FILE     := glib-networking-$($(PKG)_VERSION).tar.xz
+$(PKG)_URL      := https://download.gnome.org/sources/$(PKG)/$(call SHORT_PKG_VERSION,$(PKG))/$($(PKG)_FILE)
+$(PKG)_DEPS     := cc gnutls glib
+
+define $(PKG)_UPDATE
+   $(WGET) -q -O- 'https://git.gnome.org/browse/glib-networking/refs/tags' | \
```

Comment:
This update does not work. try this:
    $(WGET) -q -O- 'https://gitlab.gnome.org/GNOME/glib-networking/-/tags' | \
    $(SED) -n "s,.*glib-networking-\([0-9]\+\.[0-9]*[0-9]*\.[^']*\)\.tar.*,\1,p" | \
    $(SORT) -Vr | \